### ISSUE TITLE
feature: show output of tests

### ISSUE THREAD

**Issue by steffahn** [rust-lang/cargo-bisect-rustc#205]
I often use the `--prompt` option and then hold down the return key for a while because the (pre-selected) automatic analysis based on `--regress` or a script ***is correct***, yet I want to see the actual output to double check that e.g. there’s no other ICE or error sneaking in or whatnot. I feel like there should be a flag for showing the output without such workarounds. (Unless I’m missing the fact that there might already be such an option.)

**Comment by ehuss** [comment]
`-vv` should show the output. Does that work for you?

**Comment by steffahn** [comment]
I’ll test that shortly…

**Comment by steffahn** [comment]
Yes it does work, thanks :-) But that should probably be better documented. (Also I didn’t quite see yet what `-v` does compared to no verbosity level at all.)

**Comment by ehuss** [comment]
Another minimum thing would be to always show the output if the "start" fails. That often happens when either you pick the wrong start point, or your test itself is broken. Either way, the output is very important.

Unfortunately the logic around capturing the output is a little complicated.

**Comment by RalfJung** [comment]
> `-vv` should show the output. Does that work for you?

Oh wow, that's a well-hidden feature. I tried `-v` to no avail. I didn't expect that repeating the `-v` would make any difference...
